spring-ai/src/main/resources/application.yml

```diff
@@ -0,0 +1,3 @@
+spring:
+  ai:
+    openai.api-key: ${OPEN_AI_KEY}
```

Comment:
minor: aren't we missing an empty line after? for lint verification purposes.. not sure though